ietf/templates/doc/document_ballot_content.html

```diff
@@ -39,7 +39,7 @@
                 </p>
             {% endif %}
         {% endif %}
-        {% if not ballot_open %}
+        {% if not ballot_open or iesg_state == "ann" or iesg_state == "approved" %}
```

Comment:
I'm a little uncomfortable that this now doesn't tell the truth - While in 'approved", the ballot is, in reality, still open as far as the datatracker is concerned, but the IESG will not make more changes to positions.It is in the transient phase waiting for the secretariat to clear, hence close, it at the time of announcement.
But the subtlety isn't really useful to the consumer of this message, so we'll go with this as is.